resources/assets/js/components/partials/form/Select.vue

```diff
@@ -12,63 +12,85 @@ select:focus {
 
 <template>
   <div>
-    <label :for="id" class="mb2" :class="{ b: required }" v-if="title">{{ title }}</label>
+    <label v-if="title" :for="id" class="mb2" :class="{ b: required }">
+      {{ title }}
+    </label>
     <select
-        :value="selectedOption"
-        @input="event => { $emit('input', event.target.value) }"
-        :id="id"
-        :name="id"
-        required
-        :class="formClass != null ? formClass : 'br2 f5 w-100 ba b--black-40 pa2 outline-0'">
-        <option v-for="option in options" :key="option.id" :value="option.id" v-if="option.id != excludedId">{{ option.name }}</option>
+      :id="id"
+      :value="selectedOption"
+      :name="id"
+      required
+      :class="formClass != null ? formClass : 'br2 f5 w-100 ba b--black-40 pa2 outline-0'"
+      @input="event => { $emit('input', event.target.value) }"
+    >
+      <option v-for="option in filterExclude(options)" :key="option.id" :value="option.id">
```

Comment:
Here I've remove ``v-if``, because it should not be used with ``v-for``. It's been replaced by the `filterExclude` function.